library/opensuse

```diff
@@ -4,22 +4,16 @@
 Maintainers: Flavio Castelli <fcastelli@suse.com> (@flavio),
              Aleksa Sarai <asarai@suse.com> (@cyphar),
              Jordi Massaguer Pla <jmassaguerpla@suse.com> (@jordimassaguerpla),
-             David Cassany Viladomat <dcassany@suse.com> (@davidcassany)
+             David Cassany Viladomat <dcassany@suse.com> (@davidcassany),
+             Fabian Vogt <fvogt@suse.com> (@Vogtinator)
 GitRepo: https://github.com/openSUSE/docker-containers-build.git
 Constraints: !aufs
 
-# This is the multiarch section specific for openSUSE-Leap-42.2
-Tags: 42.2
-Architectures: amd64
-GitFetch: refs/heads/openSUSE-Leap-42.2
-GitCommit: fd39ce9631c3b6a110b6c1e8e5756a20d0863a77
-amd64-Directory: x86_64/
-
 # This is the multiarch section specific for openSUSE-Leap-42.3
 Tags: 42.3,latest,leap
 Architectures: amd64, arm64v8, ppc64le
 GitFetch: refs/heads/openSUSE-Leap-42.3
-GitCommit: 6252e7bc2a411b3c5cfd5ed3b74e5839f7f7615b
+GitCommit: 1b3683775818c7d5f7fe4c7b60781a35edd2b7b4
```

Comment:
https://github.com/openSUSE/docker-containers-build/commit/81638d69fffe52e00863e83e9949a0c5d2cfdba3 is a commit which modifies tarballs -- given that Git doesn't handle tarballs in commit history well (ballooning repository size), we ask that base image maintainers rebase or amend their Git commit history such that only a single commit ever includes tarballs. :pray: :heart:

(In case you're curious, see https://github.com/docker-library/official-images/issues/1095 for details about why Git LFS, which is a common solution to this problem, doesn't currently work for our use case. :disappointed:)

Reply:
Yes, that's normally also the case for our script.
In this case I opted to do a pull request instead to test the automation. It's not possible to diff a commit without history, you know :-)

If that needs to be changed, I can delete the branch history.

Reply:
We diff the before/after Docker build contexts directly, but I can see where the value in the PR upstream was.  If you don't mind deleting the history now that the PR's been made and reviewed, that'd be great.  Thanks! :+1:

Reply:
I'll do so tomorrow and edit this PR accordingly.